### ISSUE TITLE
Product 'PPRiskMagnes' not found in package 'Braintree'. it is required by package 'BraintreeDropIn' target 'BraintreeDropIn'.

### ISSUE THREAD

**Issue by RamblinWreck77** [braintree/braintree-ios-drop-in#337]
- SDK/Library Version: Braintree 5.4.2, Drop in 9.0.2
 - Environment: All
 - iOS Version: 14.6
 - Device: All
 - Integration type & version [e.g. CocoaPods 1.1.1, Carthage 0.37.0, Swift Package Manager 5.1]

Bug: Can't build w/error:
```
product 'PPRiskMagnes' not found in package 'Braintree'. it is required by package 'BraintreeDropIn' target 'BraintreeDropIn'.
```

1. Make a project
2. Add Braintree and braintree iOS dropin via SPM
3. See error when you compile

https://github.com/braintree/braintree_ios/releases/tag/5.4.2

5.4.2 seems to have removed PPRiskMagnes, which drop in is expecting/depends on. Is there any work around?

**Comment by scannillo** [comment]
👋  Hi @RamblinWreck77 - the removal of the PPRiskMagnes framework in `braintree_ios` is part of a bug-fix provided by Xcode in Xcode 12.5. We will be making a DropIn release very soon to use this newer version of `braintree_ios`. 

However - until that release, you should continue using **maximum** `braintree_ios` 5.4.1 with `braintree-ios-drop-in` 9.0.2.

**Comment by RamblinWreck77** [comment]
Setting the max version to 5.4.1 worked :) Thanks!

**Comment by scannillo** [comment]
👋 Hi @RamblinWreck77 - this was fixed in[ Braintree iOS Drop In v9.1.0. ](https://github.com/braintree/braintree-ios-drop-in/releases/tag/9.1.0)
